internal/compiler/widgets/material/std-widgets-impl.slint

```diff
@@ -9,7 +9,7 @@ import { MaterialPalette } from "styling.slint";
 export { Switch } from "switch.slint";
 export { ListItem } from "components.slint";
 export { LineEdit } from "lineedit.slint";
-
+export { MenuBarItem, MenuBar, MenuFrame, MenuItem } from "menu.slint";
```

Comment:
This makes `MenuFrame` part of the std-widgets API. Is this intentional? If yes, then I think documentation is missing.

Reply:
Thanks! I completely overlooked that :)